### DISCUSSION TITLE
Adding feature to embed code snippet while writing content in WYSIWYG editor

### DISCUSSION THREAD

**Discussion by Vprasannaakumar** [directus/directus#11858] [Feature Requests]
Currently WYSIWIG editor doesn't showing option to embed code like programming languages like java, python etc while writing content.  In markdown editor, by default it is showing. It would be helpful if it is present in WYSIWIG editor also.

**Comment by rijkvanzanten** [comment]
The difficulty with doing a code block in a HTML-based wysiwyg is that the output HTML is going to be very opinionated. For example, integrating something like TinyMCE's code plugin means that you're required to use prism.js on your output website. Markdown doesn't have this problem, as the codeblock language is part of the language itself.

**Comment by metatronicx** [comment]
Any update on this ? Tech writers often needs this. Would be great if we can have it in directus

**Reply by rijkvanzanten** [reply]
We prioritize based on community feedback, and this hasn't really gotten enough community engagement or upvotes for us to look at this to include in core. For now, we recommend using the markdown editor as that already does code blocks.

**Reply by metatronicx** [reply]
Ok @rijkvanzanten gonna try the markdown

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
